dnf-behave-tests/dnf/upgrade.feature

```diff
@@ -216,16 +224,20 @@ Examples:
         | --enhancement |
 
 
-# @dnf5
-# TODO(nsella) update command alias not supported, stderr differs
-@bz1585138
-Scenario Outline: Print correct number of available updates if update <type> is given and updateinfo is available
+@dnf5
+# dnf4 @bz1585138
+# dnf5: we intentionally do not offer unrequested info
+Scenario Outline: Do nothing when updateinfo is available but other than requested <type> update is available
   Given I use repository "dnf-ci-fedora-updates"
     And I use repository "dnf-ci-thirdparty-updates"
    When I execute dnf with args "update <type>" 
    Then the exit code is 0
     And Transaction is empty
-    And stderr contains "No security updates needed, but 7 updates available"
```

Comment:
I believe we actually want this: https://github.com/rpm-software-management/dnf5/issues/360
I would keep the test disabled.

Reply:
OK, I've discussed this personally with Jarda and he said it's not necessary to provide unrequested info, especially a number that says almost nothing (just there can be complaints that it is incorrect). I'd say - if necessary - just info that "other types of updates are available" will do. However, the scenario has been disabled as requested.